### PR TITLE
[native] Remove sleep time from query app

### PR DESCRIPTION
**Pull request by gichan-jang**
During sleep, query server cannot hanlde event.
Remove sleep and use g_main_loop.

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: native/example_query_object_detection/example_query_object_detection.c [hunk 1]
```diff
@@ -10,6 +10,23 @@
 #include <glib.h>
 #include <nnstreamer.h>
 
+GMainLoop *loop; /**< main event loop */
```
Comment (jaeyun-jung) [inline native/example_query_object_detection/example_query_object_detection.c:13]: add static :)

Comment (gichan-jang) [reply native/example_query_object_detection/example_query_object_detection.c]: Updated.

**Review by anyj0527 — APPROVED** [review]
LGTM ✔

File: native/example_query_object_detection/example_query_object_detection.c [hunk 2]
```diff
@@ -10,6 +10,23 @@
...
+static gboolean
+timeout_cb (gpointer user_data)
+{
+  g_message ("timeout_cb");
+  if (loop) {
+    g_main_loop_quit (loop);
+    g_main_loop_unref (loop);
+    loop = NULL;
+  }
+  return TRUE;
```
Comment (jaeyun-jung) [inline native/example_query_object_detection/example_query_object_detection.c:27]: well..recommend return false if you dont want to set timer repeatedly.

Comment (gichan-jang) [reply native/example_query_object_detection/example_query_object_detection.c]: Right! It works only once, so it's better to set it to `FALSE`.
Thanks.

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**